### DISCUSSION TITLE
需求建议：点击通知后打开App增加自动操作

### DISCUSSION THREAD

**Discussion by axelburks** [chanify/chanify#46] [Ideas]
例如：
设置自动复制的文字类型的通知，点击通知后，打开App，自动复制文字到剪贴板（目前是需要下滑或长按通知才能复制）
链接类型的通知，可通过参数设置自动打开，此时点击通知打开App后自动打开对应链接（包括 scheme 类型）

**Comment by wizjin** [comment]
谢谢你的建议，后续我会考虑如何处理相关的交互会比较好。
